improver_tests/regrid/test_AdjustLandSeaPoints.py

```diff
@@ -45,7 +45,6 @@
     set_up_variable_cube,
 )
 from improver.utilities.spatial import OccurrenceWithinVicinity
-from improver.utilities.warnings_handler import ManageWarnings
 
 # The warning messages are internal to the iris.analysis module v2.2.0
 IGNORED_MESSAGES = ["Using a non-tuple sequence for multidimensional indexing"]
```

Comment:
`IGNORED_MESSAGES` and `WARNING_TYPES` are no longer used and need to be deleted.